associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

```diff
@@ -82,7 +83,7 @@
     <choose>
       <if type="chapter entry-dictionary entry-encyclopedia">
         <names variable="container-author" delimiter=", ">
-          <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
+          <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="always">
```

Comment:
is the removal of `initialize-with` here intentional? It's used everywhere else in the style

Reply:
Thanks, @adam3smith! 
It was an error. Can you fix it for us or must I submmit another pull request?

Reply:
I can fix